corehq/util/soft_assert/api.py

```diff
@@ -16,7 +16,7 @@ def _send_message(info, backend):
     )
 
 
-def soft_assert(to, notify_admins=False,
+def soft_assert(to=None, notify_admins=False,
```

Comment:
based on the rest of the function it looked like it was intended to be supported with `to` being empty, so it seemed fine to make it an optional argument.